lmdeploy/vl/model/builder.py

```diff
@@ -30,40 +33,21 @@ def load_vl_model(model_path: str,
         model_path = get_model(model_path,
                                revision=revision,
                                download_dir=download_dir)
+
+    max_memory = None
+    if not with_llm:
+        import torch
+        tp = getattr(backend_config, 'tp', 1)
+        max_memory = {i: torch.cuda.mem_get_info(i)[0] for i in range(tp)}
+
+    kwargs = dict(with_llm=with_llm, max_memory=max_memory)
```

Comment:
We can put `config` into `kwargs`, so that derived classes like `LlavaVisionModel` doesn't need to `get_hf_config_content` again